app/webpack/stylesheets/components/_table.scss

```diff
@@ -197,3 +197,14 @@ td.active {
   background: govuk-colour("light-grey");
   text-align: center;
 }
+
+/* Ensures the table has full width and no margin at the bottom */
+.govuk-table--custom {
+  margin-bottom: 0;
```

Comment:
It might be possible to use the GDS spacing helpers (https://design-system.service.gov.uk/styles/spacing/) instead of creating custom CSS.